config.yml

```diff
@@ -23,7 +23,7 @@ destination:
   rabbitmq_consumer_connection:
     user: guest
     password: guest
-    host: localhost
```

Comment:
I suggest to keep it as localhost for default  and where we want to change it we use environment variables.

Reply:
it might be a bit challenging for developers to understand this. 
I was struggling with this too.